source/includes/_quick_guide.md

```diff
@@ -1,27 +1,28 @@
 ## Quick Guide
 
+### Management
+
+* Each user in the Metislabs API will belong to an `organisation`.
+* Each `organisation` can have one or more `sites` associated with the `organisation`, in most cases, `sites` individual factories.
+* Each `site` contains multiple `processes` which may have `forecasts` associated with them.
+* Trying to access `organisation`, `site`, `proccess` and `forecast` which are not authorised to the user will result in a
```

Comment:
I think user should be replaced with token in this section so as not to cause confusion with actual users